app/models/manageiq/providers/openstack/inventory/collector/target_collection.rb

```diff
@@ -309,7 +311,7 @@ def infer_related_vm_ems_refs_db!
 
       all_stacks.each { |s| add_simple_target!(:orchestration_stacks, s.ems_ref, :tenant_id => s.cloud_tenant.ems_ref) }
       vm.cloud_networks.collect(&:ems_ref).compact.each { |ems_ref| add_simple_target!(:cloud_networks, ems_ref) }
-      vm.floating_ips.collect(&:address).compact.each { |address| add_simple_target!(:floating_ips, address) }
+      vm.floating_ips.collect(&:ems_ref).compact.each { |address| add_simple_target!(:floating_ips, address) }
```

Comment:
probably rename also the other address to ems_ref

Reply:
:+1: